companies-manifest.js

```diff
@@ -853,6 +853,40 @@ const companies = [
       },
     },
   },
+  {
+    key: 'https://wazirx.com/',
+    similarWebGlobalRank: 48011,
+    name: 'WazirX',
+    logo: null,
+    style: {},
+    projects: {
+      taskade: {
```

Comment:
The keys on the projects need to represent the project itself.
It's used in the URL for its showcase page otherwise the project's showcase page won't be able to render.